### DISCUSSION TITLE
环境变量设置无效

### DISCUSSION THREAD

**Discussion by sacrrie** [ChatGPTNextWeb/NextChat#1820] [Q&A] [answered]
不管是Docker容器还是Vercel一键部署，我输入的环境变量：code和api key 都不起效果，载入后并不会提示我输入密码，而且不管是否在设置中心输入code 和API key 都无法正常使用，一直提示没有quota 。不知道是我看漏了什么还是作者本身不打算让人继续用其project.

**Comment by sacrrie** [comment, marked as answer]
没有去分清API Key和ChatGPT Plus 的区别，Plus会员和API使用量没有关系，敬请注意。

**Comment by kele527** [comment]
我设置了code也无效，没有让输入密码
